### DISCUSSION TITLE
Image Carousel or Static image for home page

### DISCUSSION THREAD

**Discussion by vivekkpatel** [cotes2020/jekyll-theme-chirpy#2049] [Ideas]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### Is your feature request related to a problem? Please describe

I am trying to add static image to main page, i have to modify home.html layout for this

### Describe the solution you'd like

 it's common feature that anyone can use it's better to add this to config file 
 additionally if we can automatically choose images based on light / dark theme, that would be just perfect

### Describe alternatives you've considered

i did this modification on my html page and added image path on index.html
```html
{% if page.image %}
{% capture src %}src="{{ page.image.path | default: page.image }}"{% endcapture %}
{% capture class %}class="preview-img{% if page.image.no_bg %}{{ ' no-bg' }}{% endif %}"{% endcapture %}
{% capture alt %}alt="{{ page.image.alt | xml_escape | default: "Preview Image" }}"{% endcapture %}

{% if page.image.lqip %}
{%- capture lqip -%}lqip="{{ page.image.lqip }}"{%- endcapture -%}
{% endif %}

<div class="mt-3 mb-3">
    <img {{ src }} {{ class }} {{ alt }} w="1200" h="630" {{ lqip }}>
    {%- if page.image.alt -%}
    <figcaption class="text-center pt-2 pb-2">{{ page.image.alt }}</figcaption>
    {%- endif -%}
</div>
{% endif %}
```

**Comment by kungfux** [comment]
I think we cannot say that this is an essential feature that many people will find useful. Also, there is no single solution that will suit everyone, for example, some people prefer to place the background image at the bottom of the page like grass, others at the top like clouds, a third option is to make a repeatable background. It's better to keep it custom, which gives you as much room for creativity as you need, while keeping only the core and necessary features in the codebase, as every feature has its maintenance costs. Moreover, this does not even require you to change the code, as this change can be achieved by adding an extra CSS rule, I believe.
